src/content/docs/get-involved.md

```diff
@@ -0,0 +1,25 @@
+---
+title: Get involved
+---
+
+We appreciate your interest in contributing to Interledger and welcome contributions from everyone who shares our vision of an open, fair, and financially inclusive web. It is important that we maintain a safe and positive community experience for all, so we do require that all who are willing to participate agree to adhere to our [Participation Guidelines](/participation-guidelines).
+
+## Contributing to Interledger
+
+If you are new to Interledger and are not sure where your skills can be applied, we hope that the following list can help you get started. And we welcome your questions and feedback in [our Slack channel](https://communityinviter.com/apps/interledger/interledger-working-groups-slack).
+
+If you have experience with maintaining and contributing to **open-source projects**, we would love to have you (pending text to be written).
+
+If you are most comfortable working with **Typescript**, you can contribute to [Rafiki](https://github.com/interledger/rafiki), [Testnet](https://github.com/interledger/testnet) or the [Open Payments SDK](https://github.com/interledger/open-payments).
+
+If you are most comfortable working with **React**, you can contribute to [Rafiki](https://github.com/interledger/rafiki) or [Testnet](https://github.com/interledger/testnet).
+
+If you are most comfortable working with **Angular**, you can contribute to [Testnet](https://github.com/interledger/testnet) or [Interledger Pay](https://github.com/interledger/interledger-pay).
```

Comment:
I don't think we have any Angular projects, all of the frontends are react-based

Reply:
This list came from @AlexLakatos though 🤔